### ISSUE TITLE
Bug - LocalStore: cannot set store permissions 0744 (EPERM: operation not permitted, chmod '/app/uploads')

### ISSUE THREAD

**Issue by karlos-silva2** [RocketChat/Rocket.Chat#24258]
Erro:

I have this permission problem in the uploads folder

![image](https://user-images.githubusercontent.com/85645175/150597486-533b5163-931c-46b2-9157-816c43a419b0.png)

Rocket.Chat Version: 4.3.2 |
NodeJS Version: 12.22.1 - x64 |
MongoDB Version: 3.6.12 |
Platform: linux |
Process Port: 3000 |
ReplicaSet OpLog: Enabled |
Commit Hash: 4e8ba15 |
Commit Branch: HEAD

**Comment by dudanogueira** [comment]
Hi!

This is probably not a software bug. You need to set the permissions accordingly in your deployment.

**Comment by d3hof** [comment]
Sorry for commenting on an old issue. So I observed that issue on kubernetes with ceph-fs, where for some reason the directory always gets mounted with ownership root:root. We are not running rocket.chat in priviliged mode and I only can observer this problem, on an instance where we activated the Poll app https://github.com/sampaiodiego/rocket.chat.app-poll. It seems like for some reason it runs some change of ownership on the directory /app/uploads which is mounted via cephfs volumeMount.
